docs/reference/snapshot-restore/apis/create-snapshot-api.asciidoc

```diff
@@ -0,0 +1,193 @@
+[[create-snapshot-api]]
+=== Create snapshot API
+++++
+<titleabbrev>Create snapshot</titleabbrev>
+++++
+
+Takes a <<snapshot-restore,snapshot>> of a cluster or specified data streams and
+indices.
+
+////
+[source,console]
+-----------------------------------
+PUT /_snapshot/my_repository
+{
+  "type": "fs",
+  "settings": {
+    "location": "my_backup_location"
+  }
+}
+-----------------------------------
+// TESTSETUP
+////
+
+[source,console]
+-----------------------------------
+PUT /_snapshot/my_repository/my_snapshot
+-----------------------------------
+// TEST[s/my_snapshot/my_snapshot?wait_for_completion=true/]
+
+[[create-snapshot-api-request]]
+==== {api-request-title}
+
+`PUT /_snapshot/<repository>/<snapshot>`
+
+`POST /_snapshot/<repository>/<snapshot>`
+
+[[create-snapshot-api-desc]]
+==== {api-description-title}
+
+You can use the create snapshot API to create a <<snapshot-restore,snapshot>>, which is a
+backup taken from a running {es} cluster.
+
+By default, a snapshot includes all data streams and open indices in the
+cluster, as well as the cluster state.  You can change this behavior by
+specifying a list of data streams and indices to back up in the body of the
+snapshot request.
+
+NOTE: You must register a snapshot before performing snapshot and restore operations. Use the <<put-snapshot-repo-api,put snapshot repository API>> to register new repositories and update existing ones.
+
+The snapshot process is incremental. When creating a  snapshot, {es} analyzes the list of files that are already stored in the repository and copies only files that were created or changed since the last snapshot. This process allows multiple snapshots to be preserved in the repository in a compact form.
```

Comment:
Extra space. Line wrapping is also too long.
```suggestion
The snapshot process is incremental. When creating a snapshot, {es} analyzes the list of files that are already stored in the repository and copies only files that were created or changed since the last snapshot. This process allows multiple snapshots to be preserved in the repository in a compact form.
```